Userland/Libraries/LibWeb/CSS/Length.h

```diff
@@ -38,6 +38,7 @@ class Length {
 
     // We have a RefPtr<CalculatedStyleValue> member, but can't include the header StyleValue.h as it includes
     // this file already. To break the cyclic dependency, we must move all method definitions out.
+    Length();
```

Comment:
This feels a bit nasty, is this only done so the `Length params[4];` in the parser works?
If so it'd make more sense to make that `Vector<Length, 4>` and append the Lengths (which avoids needing a default ctor).